core/src/main/scala/org/apache/spark/internal/config/package.scala

```diff
@@ -287,4 +287,10 @@ package object config {
       .bytesConf(ByteUnit.BYTE)
       .createWithDefault(100 * 1024 * 1024)
 
+  private[spark] val REDUCER_MAX_REQ_SIZE_SHUFFLE_TO_MEM =
+    ConfigBuilder("spark.reducer.maxReqSizeShuffleToMem")
+      .doc("The blocks of a shuffle request will be fetched to disk when size of the request is " +
+        "above this threshold. This is to avoid a giant request takes too much memory.")
+      .longConf
+      .createWithDefault(200 * 1024 * 1024)
```

Comment:
Could I use `bytesConf(ByteUnit.BYTE)` ?  Thus I don't need to convert it to bytes by `ByteUnit.MiB.toBytes(xxx).toLong`